src/runtime_src/core/edge/drm/zocl/cu_scu.c

```diff
@@ -223,19 +223,26 @@ void xrt_cu_scu_fini(struct xrt_cu *xcu)
 		put_pid(p);
 		goto skip_kill;
 	}
-
+	
 	ret = kill_pid(p, SIGTERM, 1);
 	if (ret) {
 		DRM_WARN("Failed to terminate SCU pid %d.  Performing SIGKILL.\n",core->sc_pid);
 		kill_pid(p, SIGKILL, 1);
 	}
+	// Wait for process with pid to finish
+	p = find_get_pid(core->sc_pid);
+	while(p) {
```

Comment:
space after while 
while (p)

Reply:
Also, I am wondering if we really need this while loop? Shouldn't kill_pid make sure the process is killed? And if looking at line 237, are we put_pid(NULL) here?